command/polly_tts.js

```diff
@@ -5,6 +5,10 @@ var audioconcat = require('audioconcat');
 var glob = require('glob');
 const logger = require('../logger');
 const xcodeQueue = require('./xcodeQueue');
+const ffmpeg = require('fluent-ffmpeg');
+const s3 = new AWS.S3({ apiVersion: '2006-03-01' });
+const Utils = require('./utils');
+const utils = new Utils();
```

Comment:
I'm wondering about adding s3 & ffmpeg dependencies to this file. I think I would expand Utils.js into FileUtils and include the new functions there instead so this one remains focused on Polly.

Reply:
Yes, I'll file a follow up for this one.  Good call.

Reply:
https://github.com/MozScout/scout-ua/issues/157